.github/workflows/build_and_test.yml

```
@@ -0,0 +1,203 @@
+    env:
+      MIX_ENV: prod
+    steps:
+      <<: *checkout-cache
+      - name: Common Versioning
+        uses: ./.github/actions/common-versioning
+        id: version
+        with:
+          mode: automatic
+      <<: *node-setup
+      <<: *docker-build
+      # Add additional automatic steps as needed (pushing, etc.) 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)